src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { PeopleContext } from '../../peopleContext';
+import { getPersonByName } from '../../utils/getPeopleByName';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug: userId, sex, born, died, fatherName, motherName } = person;
+  const { slug: selectedUserId } = useParams();
+
+  const people = useContext(PeopleContext);
+  const mother = getPersonByName(motherName, people);
+  const father = getPersonByName(fatherName, people);
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': userId === selectedUserId,
```

Comment:
The class name 'has-background-warning' is applied when `userId` equals `selectedUserId`. This logic is fine, but the class name itself could be more descriptive. Consider renaming it to something more meaningful, such as 'selected-person', to make the code more readable.